# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xanim/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- xanim2801/Makefile.vanilla	2005-10-09 12:43:32.000000000 +0200
+++ xanim2801/Makefile	2005-10-09 12:44:11.000000000 +0200
@@ -296,8 +296,8 @@
 # XLIBDIR = -L/usr/openwin/lib
 #
 ###-- Linux using X11R6
-INCLUDE =  -I/usr/X11R6/include -I/usr/X11R6/include/X11
-XLIBDIR = -L/usr/X11R6/lib
+INCLUDE =  -I/usr/X11/include -I/usr/X11/include/X11
+XLIBDIR = -L/usr/X11/lib
 #
 ###-- My SunOS machine running X11R5
 # INCLUDE = -I/usr/X11R5/include -I/usr/X11R5/include/X11
